testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4777,6 +4777,23 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'cancel button is focused');
     });
 
+    QUnit.testInActiveWindow('Popup has open state if tab key was pressed when applyValueMode: "instantly"', function(assert) {
```

Comment:
```suggestion
    QUnit.testInActiveWindow('Popup should be not closed if tab key was pressed when applyValueMode: "instantly"', function(assert) {
```

Reply:
Sounds good! Thank you! Fixed in [99a178d2a43ec5be8ad8dd319a440caaa9118077].